examples.md

```diff
@@ -379,6 +380,17 @@ Replace `~/.cache/pip` with the correct `path` if not using Ubuntu.
       ${{ runner.os }}-pip-
 ```
 
+## Python - pipenv
```

Comment:
Thanks for adding this example! Can you please also link to this from the README's [Implementation Examples section](https://github.com/actions/cache#implementation-examples)?

Reply:
Done! 👍

Reply:
Thank you!